app/build.gradle

```diff
@@ -88,6 +90,8 @@ dependencies {
     implementation 'com.twitter.sdk.android:tweet-ui:3.1.1'
 
     // Annotations
+    implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
```

Comment:
Refused --> use com.google.materiallayout equivalent, following guidelines here https://developer.android.com/jetpack/androidx/migrate

Reply:
Yep ! sry don't know how these lines came. These are useless. Thanks for notifying !